python/lsst/afw/cameraGeom/utils.py

```diff
@@ -730,7 +730,11 @@ def makeImageFromCamera(camera, detectorNameList=None, background=numpy.nan, buf
 
     boxList = getCcdInCamBBoxList(ccdList, binSize, pixelSize_o, origin)
     for det, bbox in zip(ccdList, boxList):
-        im = imageSource.getCcdImage(det, imageFactory, binSize)[0]
+        try:
+            im = imageSource.getCcdImage(det, imageFactory, binSize)[0]
+        except Exception as exc:
+            print("Unable to get image for detector %s: %s" % (det.getName(), exc))
```

Comment:
I assume no logger is available, but shouldn't this go to `sys.stderr`?

Reply:
This follows the existing pattern. Would you like me to change that here?